### ISSUE TITLE
Joint2D element Errors

### ISSUE THREAD

**Issue by MelakuSeyoum** [zhuminjie/OpenSeesPyDoc#88]
Greetings Minjie ,

I am trying to model a simple 2D frame with the joint properties modeled using the Joint2D element. I followed the syntax & procedures written in the documentation. 

`element('Joint2D', eleTag, *eleNodes, [Mat1, Mat2, Mat3, Mat4, ]MatC, LrgDspTag)`

Neglected the optional materials for the rotational springs at nodes [Mat1, Mat2, Mat3, Mat4, ] and used the following format:

`element('Joint2D', eleTag, *eleNodes, MatC, LrgDspTag)`

where eleNodes is an array of [nd1,nd2,nd3,nd4,ndC]. ndC is not used anywhere else in the model. See the code excerpt below.
```
eleTag  = 101
eleNodes = [11001, 11002, 11003, 11004, 11000]
matTagWP = 4      #simple elastic uni-axial material for check
LrgDspTag = 0      #for small deformations and constant geometry
element('Joint2D', eleTag, eleNodes, matTagWP, LrgDspTag)
```
But when I run analysis, i get an error:
```
  File "C:/OP/PortalFrame2d Analysis.py", line 394, in <module>
    element('Joint2D', eleTag, eleNodes, matTagWP, LrgDspTag)
OpenSeesError: See stderr output
```
I'm using OpenSeesPy 0.5.3 and Anaconda 2019 (with Spyder). Would you comment on the issue? Is it a bug or have I made a mistake in the code? Thanks in advance!

**Comment by zhuminjie** [comment]
you have to call it as (star is missing)

element('Joint2D', eleTag, *eleNodes, matTagWP, LrgDspTag)

**Comment by MelakuSeyoum** [comment]
Thank you for your response. I had actually tried putting the asterisk mark (even before reporting the issue.), however, this is not solving the problem. In a bid to easily identify the problem, I once again tried to use the Joint2D element for a simplified single beam-to-column joint as shown in the sketch below. Yet the code terminates with the following error message: 
```
There is a bug in Joint2D constructor -- fix later
Traceback (most recent call last):
....
....
....
  File "C:/OP/PortalFrame2d v17.py", line 394, in <module>
    element('Joint2D', eleTag, *eleNodes, matTagWP, LrgDspTag)

OpenSeesError: See stderr output
```
As is displayed in the error report, perhaps there is a bug. Would you please check? Thank you once again!
![Diagram2](https://user-images.githubusercontent.com/49725625/67681843-0b2dde00-f986-11e9-9952-8a1d37b1dbba.png)

**Comment by zhuminjie** [comment]
Is your parameter same as you initially gave?

eleTag  = 101
eleNodes = [11001, 11002, 11003, 11004, 11000]
matTagWP = 4      #simple elastic uni-axial material for check
LrgDspTag = 0      #for small deformations and constant geometry

**Comment by MelakuSeyoum** [comment]
Yes, it is. And, I have made sure that node "11000" was not defined/used prior to this call.

**Comment by zhuminjie** [comment]
I think your syntax might be correct, this line

"There is a bug in Joint2D constructor -- fix later" is a developer backdoor. I will fix it. Thank you!

**Comment by MelakuSeyoum** [comment]
Thank you! I was looking for the source code hoping to see if I could help with the Joint2D constructor implementation in OpenSeesPy. All I was able to find was the code for the documentation. Please advise on where I could access it, if it has been made public already. Thank you once more for your prompt reply.

**Comment by zhuminjie** [comment]
https://github.com/OpenSees/OpenSees

**Comment by MelakuSeyoum** [comment]
The Joint2D element error is now resolved in OpenSeesPy version 0.5.8. Thank you!
